docs/getting-started/backends-and-brokers/kafka.rst

```diff
@@ -0,0 +1,34 @@
+.. _broker-kafka:
+
+=============
+ Using Kafka
+=============
+
+.. _broker-Kafka-installation:
+
+Configuration
+=============
+
+For celeryconfig.py:
+
+.. code-block:: python
+
+    broker_url = 'confluentkafka://localhost:9092'
+    result_backend = 'redis://localhost:6379/0'
+    broker_transport_options = {"allow_create_topics": True}
```

Comment:
1. Do you have to set this?
2. Is using `json` mandatory?

Reply:
1. No. I did at first, but I think that once the topic is created, I don't. Maybe I can note that.
2. For the `task_serializer`, yes, `json` is mandatory. For `result_serializer`, no. I'll remove that one.

Reply:
I pushed up my changes!